src/data/hicdex.js

```diff
@@ -472,16 +474,66 @@ export async function fetchObjkts(ids) {
   return data.token
 }
 
+async function fetchSubjktNames(addresses) {
+  const { errors, data } = await fetchGraphQL(
+    `query SubjktNames ($addresses: [String!]) {
+    holder(where: {address: {_in: $addresses}}) {
+      address
+      name
+    }
+  }
+  `,
+    'SubjktNames',
+    { addresses }
+  )
+  if (errors) {
+    console.error('Failed to fetch SUBJKTs')
+    console.error(errors)
+  }
+  return data.holder
+}
+
 export async function fetchObjktDetails(id) {
   const { errors, data } = await fetchGraphQL(query_objkt, 'objkt', { id })
   if (errors) {
     console.error(errors)
   }
 
-  console.log(errors, data)
-
   const result = data.token_by_pk
-  console.log(result)
+
+  // Fetch burn transfers
+  // we want to use SUBJKT to resolve aliases not tzkt.
+  const addressesToResolve = []
+  result.transfers = []
+  const endpoint = `${process.env.REACT_APP_TZKT_API}/v1/operations/transactions?target.eq=${HEN_CONTRACT_FA2}&parameter.[*].txs.[*].token_id=${data.token_by_pk.id}&parameter.[*].txs.[*].to_=${BURN_ADDRESS}&level.gte=${data.token_by_pk.level}&entrypoint=transfer&status=applied`
+  try {
+    const burn_operations = await (await fetch(endpoint)).json()
+    burn_operations.forEach((item) => {
+      if (addressesToResolve.indexOf(item.sender.address) === -1) {
+        addressesToResolve.push(item.sender.address)
+      }
+
+      result.transfers.push({
+        timestamp: item.timestamp,
+        ophash: item.hash,
+        sender: item.sender.address,
+        opid: item.id,
+        amount: _(item.parameter.value[0].txs).sumBy('amount'),
```

Comment:
i think you need to filter out the transfers where the `_to` property is not set to the burn address.

here is a random example of how the transfer entrypoint can be called:

```
parameter: {
        entrypoint: 'transfer',
        value: [
          {
            txs: [
              {
                to_: 'tz1ejpAPJZAe2boiKUGwb6cq5dckTZ8hXn2A',
                amount: '1',
                token_id: '58486',
              },
              {
                to_: 'tz1UBZUkXpKGhYsP5KtzDNqLLchwF4uHrGjw',
                amount: '10',
                token_id: '58487',
              },
            ],
            from_: 'KT1Hkg5qeNhfwpKW4fXvq7HGZB9z2EnmCCA9',
          },
        ],
      },
```